politeiawww/www.go

```diff
@@ -601,6 +603,28 @@ func _main() error {
 				return fmt.Errorf("build cache: %v", err)
 			}
 		}
+		if p.cfg.GithubAPIToken != "" {
```

Comment:
Hrm not sure this can be done too easily.  Was github api not even returning an error when it would attempt any request within github-tracker?  Just passing that error down and out for display on a updategithub request seems like the best way to go here.